kernel/arch/dreamcast/hardware/biosfont.c

```diff
@@ -345,6 +364,6 @@ uint8 *bfont_find_icon(uint8 icon) {
 
     int icon_offset = BFONT_VMU_DREAMCAST_SPECIFIC +
         (icon * BFONT_ICON_DIMEN * BFONT_ICON_DIMEN/8);
-    uint8 *fa = get_font_address();
+    uint8 *fa = syscall_font_address();
```

Comment:
As a note, here we're giving away an address for the user to read and extract the stuff but without locking the font data to fully implement that locking in the future, this API would need to be revised to not directly expose that access to the user (or clearly warn them).

Again, notes for posterity.